src/osdag/setup.py

```diff
@@ -0,0 +1,17 @@
+# setup.py
+
+from setuptools import setup
+
+setup(
+    name='osdag-cli',
+    version='0.1.0',
+    py_modules=['osdag_cli'],
+    install_requires=[
+        'Click',
+    ],
+    entry_points={
+        'console_scripts': [
+            'osdag-cli = osdag_cli:determine_module',
+        ],
```

Comment:
This may get us too many entry points. OK for testing, but eventually we want something like:
```
osdag-cli --determine-module <filename>
```